recipes/archer/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: archer
+  version: {{ version }}
+
+source:
+  url: https://github.com/will-rowe/archer/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 34390e947644ae0b239ed4fae7b74a069a11c87e4fd575e19cf3aff2cf89800b
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('go') }}
+
+test:
+  commands:
+    - archer
+    - archer --version
+
+about:
+  home: https://github.com/will-rowe/version
```

Comment:
shouldn't *version* be *archer*?

Reply:
The --version flag was just so I could check the correct version had been built - just sanity checking really. Can remove that extra command call

Reply:
I was referring to the URL, https://github.com/will-rowe/version goes to 404.

Reply:
sorry - missed that! yes, typo